### PR TITLE
bveCVX yield redirection 26.01.2022

### PR DESCRIPTION
**Pull request by Tritium-VLK**
Sets a 15 day emissions schedule for bveCVX.  Amount still needs to be set based on sale.  This 15 days is one time meaning that going forward the emissions will run from 00:00 UTC FRIDAY giving us more time.

See #26

### CODE REVIEW

File: scripts/emissions/bveCVX_special_emissions.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+DEVELOPMENT = False
+
+
+
+# UPDATE THESE VARIABLES WITH THE NEEDS FOR A NEW GUARD LAUNCH OR SIMILAR SCENARIO
+STARTING_TIME = 1643241600
+ENDING_TIME = 1644534000
```
Comment (jorijnsmit) [inline scripts/emissions/bveCVX_special_emissions.py:10]: this translates to `datetime.datetime(2022, 2, 10, 23, 0)`?

File: scripts/emissions/bveCVX_special_emissions.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+AMOUNT_TO_EMIT_DURING_PERIOD = 100
+DURATION = ENDING_TIME - STARTING_TIME
+
+
+def main():
+    safe = GreatApeSafe(registry.eth.badger_wallets.dev_multisig)
+    safe.init_badger()
+    rewards_logger = registry.eth.rewardsLogger
```
Comment (jorijnsmit) [inline scripts/emissions/bveCVX_special_emissions.py:18]: think this should be set from techops, no?

also `rewards_logger` here is just an address string, still needs to be initiated as a contract object (`safe.contract(...)` is fastest for now).

**Review by jorijnsmit — CHANGES_REQUESTED** [review]

File: scripts/emissions/bveCVX_special_emissions.py (hunk 1 above)
Comment (jorijnsmit) [reply scripts/emissions/bveCVX_special_emissions.py]: or is the idea here to have 1 hour where nothing happens..?

Comment (Tritium-VLK) [reply scripts/emissions/bveCVX_special_emissions.py]: No.  Just forgot to adjust for timezones at it was midnight CET.

File: scripts/emissions/bveCVX_special_emissions.py (hunk 2 above)
Comment (Tritium-VLK) [reply scripts/emissions/bveCVX_special_emissions.py]: correct ser.

**Comment by Tritium-VLK** [comment]
@jorijnsmit discord is down, but needs to be set to amount of bveCVX in the poli multisig after swapping all swapped cvx for bveCVX.

**Comment by Tritium-VLK** [comment]
https://docs.google.com/spreadsheets/d/1R3dBig3f13jNgjTuNhLXcGVle1gxERNNKJD-2TAWgbE/edit
image

**Review by jorijnsmit — APPROVED** [review]
amount is from bvecvx which was in politician + newly locked:
```
>>> from decimal import Decimal
>>> Decimal(7151.824281447517145778) + Decimal(4862.422279384771517847)
Decimal('12014.24656083228819625219330')
```
timestamps; adds one more day for buffer in case bribes are delayed:
```
>>> from datetime import datetime
>>> datetime.fromtimestamp(1643241600)
datetime.datetime(2022, 1, 27, 0, 0)
>>> datetime.fromtimestamp(1644537600)
datetime.datetime(2022, 2, 11, 0, 0)
```

**Review by petrovska-petro — APPROVED** [review]
LGTM!

- confirmed timestamps ending on 11th Feb 00:00 UTC
- confirmed figure from `depositFor` txs: 

https://etherscan.io/tx/0x1f15288f7b0d1dd355e7379e430fbd466f434c2b2e4635a59bc76684fc43370e
https://etherscan.io/tx/0x17d98ee2a40d670f509d67b07e2a83d059bc946eeffa63ec060a925d3d268232

also tx history from politician in cow: 

https://gnosis-protocol.io/address/0x6F76C6A1059093E21D8B1C13C4e20D8335e2909F

**Comment by petrovska-petro** [comment]
@Tritium-VLK looks like not all commits were signed 💀

**Comment by jorijnsmit** [comment]
forced merge instead of creating new pr.

**Pull request merged**